routes/newsletters.js

```diff
@@ -0,0 +1,94 @@
+import express from 'express';
+import Ghost from '../utils/api/ghost.js';
+import Post from '../utils/models/post.js';
+import Files from '../utils/data/files.js';
+import Newsletter from '../utils/newsletter.js';
+
+const router = express.Router();
+
+router.get('/', (req, res) => res.redirect('/'));
+
+router.get('/:postId', async (req, res) => {
+    const postId = req.params.postId;
+
+    if (!postId) {
+        return res.render('dashboard/newsletters', {
+            level: 'error',
+            message: 'Post Id missing!'
+        });
+    }
```

Comment:
I don't think this is necessary here afaik, we already have a redirect on `newsletter/` endpoint.